models/sessionization/segment_web_page_views__sessionized.sql

```diff
@@ -90,10 +90,27 @@ new_sessions as (
 
     select
         *,
+
```

Comment:
eww

Reply:
i think it looks less overwhelming 😂  but ok